lesson06/assignment/comedyshows.html

```diff
@@ -0,0 +1,134 @@
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.slim.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.bundle.min.js"></script>
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 06</title>
+    </head>
+    <body>
+        <header>
+          <div class="grid-container">
+                <div class="header-left">
+                    <div class="header-left-1">
+                      <h1>BOOK MY EVENT</h1>   
+                    </div>
+                    <div class="header-left-2">
+                      <form method="get">
+                        <input type="text" name="q" placeholder="search for events">
+                        <input type="submit" value="search">
+                      </form>
+                    </div>
+                </div>
+                <div class="header-right">
+                    <label>Location<br>
+                        <select name="location">
+                            <option value="Seattle">Seattle</option>
+                            <option value="Bellevue">Bellevue</option>
+                            <option value="Tacoma">Tacoma</option>
+                        </select>
+                    </label>
+                </div>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="index.html">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="Drama.html">Drama</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="comedyshows.html">Comedy shows</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="kids.html">Kids</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+            <section>
+                <div class="side-bar">
+                    <h3>Filters</h3>
+                    <nav>
+                        <ul>
+                          <li>Language
+                            <ul class="menu-1">
+                              <li class="menu-1__li">English</li>
+                              <li>Spanish</li>
+                              <li>French</li>
+                              <li>Japanese</li>
+                              <li>Korean</li>
+                            </ul>
+                          </li>
+                        </ul>
+                      </nav>
+                </div>
+            </section> 
+            <aside>
+
+                <!--accordion-->
```

Comment:
Opening each item causes as shift in content.  This is probably not the ideal use case for an accordion.  Usually you would use it for something like an FAQ section.